route/build/src/network_topo.f90

```diff
@@ -717,6 +718,148 @@ SUBROUTINE REACH_LIST(&
  IMPLICIT NONE
  ! input variables
  INTEGER(I4B)      , INTENT(IN)                 :: NRCH            ! number of stream segments
+ logical(lgt)      , intent(in)                 :: doReachList     ! flag to compute the list of upstream reaches
+ type(var_ilength) , intent(inout), allocatable :: structNTOPO(:)  ! network topology structure
+ ! output variables
+ type(var_dlength) , intent(inout)              :: structSEG(:)    ! network topology structure
```

Comment:
comment should be "reach property structure" ?

Reply:
Fixed